lib/audit_test.pm

```diff
@@ -220,4 +221,49 @@ sub rerun_fail_cases {
     }
 }
 
+# The test code kvm_svirt_arrarmor is used by two tests.
+# We need to parse log file to check the results.
+# For kvm svirt apparmor, all test cases should pass.
+# For AppArmor negative test, some cases should fail.
+sub parse_kvm_svirt_apparmor_results {
+    my $test_name = shift;
+    my $log_file = '/tmp/vm-sep/vm-sep-crack.log';
+    assert_script_run("[[ -e $log_file ]]");
```

Comment:
Why check file existing here?

Reply:
If this file doesn't exist, it means this case fail. Need to check that before parsing it.

Reply:
got it